charts/csi-powermax/charts/csireverseproxy/templates/configmap.yaml

```diff
@@ -1,7 +1,17 @@
+{{- /*
+Deprecation notice: use of reverseproxy-config ConfigMap is deprecated and will
```

Comment:
Nice use of deprecation comments throughout. Will help when we remove these parameters in a future release.